source/index.txt

```diff
@@ -39,38 +38,42 @@ System Components
 
 A complete BI system includes the following components:
 
-- **MongoDB database:** Data storage.
-- **BI Connector:** Provides a relational schema and translates SQL
+MongoDB database
+  Data storage.
+
+BI Connector
+  Provides a relational schema and translates |sql|
   queries between your BI tool and MongoDB.
-- **ODBC data source name (DSN):**
+
+ODBC data source name (DSN)
   Holds authentication and connection configuration data.
-- **BI Tool:** Data visualization and analysis.
 
-Most BI tools can import data from a :doc:`DSN
-</tutorial/create-system-dsn>`. Setting up a DSN which
-points to your |bi-short| instance allows you to use a wide range of
-BI tools and SQL clients with a single data source configuration.
+BI Tool Data visualization and analysis.
+
+Most BI tools can import data from a
+:doc:`DSN </tutorial/create-system-dsn>`. Configure a |dsn| to
+point to your |bi-short| instance. This provides you a wide range of
+BI tools and |sql| clients with a single data source configuration.
 
-Your BI tool and your DSN always run on your local desktop, but your
-database and your |bi-short| instance
-may be local, on premises, or hosted. The following sections
-describe possible configurations for a BI system.
+Your BI tool and your |dsn| always run on your local desktop, but your
+database and your |bi-short| instance may be local, on premises, or
+hosted. The following sections describe possible configurations for a
+BI system.
 
 Hosted Database and |bi-short|
 ------------------------------
 
 .. note::
 
-    If you don’t want to install and manage the |bi-short|, you can use MongoDB
-    Atlas, our fully-managed database-as-a-service. Databases hosted on Atlas
-    natively support the |bi| and make it easy to start analyzing your data
-    with your preferred BI tools. `Create an Atlas cluster to get started
-    <https://www.mongodb.com/cloud/atlas/lp/general?utm_source=bi-connector&utm_campaign=20-docs-in-20-days&utm_medium=docs>`_.
+   If you don’t want to install and manage the |bi-short|, you can use
+   {+atlas+}, our fully-managed database-as-a-service. Databases hosted
+   on {+atlas+} natively support the |bi| and make it easy to start
+   analyzing your data with your preferred BI tools.
+   :website:`Create an {+atlas+} cluster to get started <cloud/atlas/lp/general?utm_source=bi-connector&utm_campaign=20-docs-in-20-days>`.
```

Comment:
This link is malformed (missing a / before 'cloud') and should probably be updated anyway. I'm guessing that utm campaign ended a long time ago.